README.md

```diff
@@ -1,86 +1,86 @@
-# BoldBI Embedding NodeJS Samples
+ # Bold BI Embedded Sample in NodeJS
 
-This Bold BI Node.js sample contains the Dashboard embedding sample. This sample demonstrates the dashboard rendering available in your Bold BI server.
+This project was created using NodeJS 12.13. The purpose of this application is to demonstrate how to render the dashboard available on your Bold BI server.
```

Comment:
This project was created using NodeJS 12.13. This application aims to demonstrate how to render the dashboard available on your Bold BI server.

Reply:
Content updated.